tests/bats/validate/govcms-validate-modules.bats

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bats
+# shellcheck disable=SC2002,SC2031,SC2030,SC2155
+
+load ../_helpers_govcms
+
+@test "No exported config files." {
+  export GOVCMS_CONFIG_FOLDER_PATH="tests/bats/validate/fixtures/config"
+  export GOVCMS_CORE_EXTENTION_FILE_NAME=""
+  export GOVCMS_DISALLOWED_MODULES="update
```

Comment:
I don't think we can do multiline variables in Lagoon like this - so we might need to use a delimiter and `ifs` to iterate there.

Reply:
Done. 👍